app/services/katello/registration_manager.rb

```diff
@@ -183,6 +185,10 @@ def destroy_host_record(host_id)
         Rails.logger.warn("Attempted to destroy host %s but host is already gone." % host_id)
       end
 
+      def pulp2_supported?
+        SmartProxy.pulp_master.nil? || SmartProxy.pulp_master.has_feature?(SmartProxy::PULP_FEATURE)
```

Comment:
For my understanding: if pulp_master is nil, why does that imply Pulp 2 is supported?

Reply:
honestly, i'm not sure, lets get rid of it